### PR TITLE
Improvement - events tab content

### PR DESCRIPTION
**Pull request by mmexvr**
**Describe pull-request**  
_Describe what the pull-request is about_

- Content for events tab
- Change the max(1584px > ) grid using cards to show 4 instead of 3 in a one row
- Some automatic prettier clean up

Mobile is the most important part, because people will scan our QR code

1. Go to event page
2. Look in different screen sizes
3. Try out links (POWERPOINT NOT ADDED)

Remember: 
Everything here is based on what we have in the cms already, to save time. Next event if we plan more a head, design even better page and develop it.

### CODE REVIEW

**Comment by theJohnnyMe** [comment]
I have tested on my local, page and content looks fine, links work. Found only one small thing that I mentioned in separate comment.

**Comment by theJohnnyMe** [comment]
PS: Once review is done, see if you can use option "Squash and merge" when merging to come to one commit in git log.

File: src/app/components/card/card.component.ts [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
+  constructor(private router: Router) {}
 
-  clickCard(){
-    const cardUrl = this.card['url'];
-    if(this.card['type'] === 'linkInternal') {
+  clickCard() {
+    const cardUrl = this.card["url"];
+    if (this.card["type"] === "linkInternal") {
       this.router.navigate([cardUrl]);
+      console.log(this.card["url"]);
```
Comment (gaonita) [inline src/app/components/card/card.component.ts:20]: we leave this console log here on purpose? 🤔

**Review by gaonita — APPROVED** [review]
Looks nice on any size of the screen I checked :) great work in short time!

File: src/app/components/card/card.component.ts (hunk 1 above)
Comment (mmexvr) [reply src/app/components/card/card.component.ts]: Removed now! 👁️  good eyes

**Pull request merged**